### ISSUE TITLE
Project status

### ISSUE THREAD

**Issue by andreasbhansen** [fabiobiondi/react-circular-menu#1]
Hey,

What's the status on this project? Has it been abandoned, or will source code at some point be released? :smiley:

**Comment by fabiobiondi** [comment]
dead.  Sorry :)
Used in a project but it's not ready to be released

**Comment by andreasbhansen** [comment]
That's too bad, the demo looks great and includes was I was looking for. Either way, good job! :)

**Comment by alecalanis** [comment]
+1 for release

**Comment by fabiobiondi** [comment]
I know guys.
I like it too but there are some bugs in some scenarios,  I have no time to complete it and especially to maintain an open source project :(
Furthermore there is now a new D3.js version (#4)  and I should update the code. 
We'll see in 2017 : )

**Comment by lbrdar** [comment]
Hi @fabiobiondi ,

any chance you release the code? Even if you don't have time for fixing those bugs or maintaining the project, it would still be a great help and I'm sure there would be a lot of people interested in improving it.

You've done a great job here and it would be a shame if it stayed just as a part-done project in the past.

Please think about it, I'm working on one smaller project right now and this menu would help me a lot.

Thank you,
Leo

**Comment by Anderson-Liu** [comment]
+1

**Comment by daquintero** [comment]
Yeah someone else can help take it further if you release the code!

**Comment by lonewolfaka9** [comment]
+1

**Comment by evanvin** [comment]
+1

**Comment by thechosenjuan** [comment]
+1

**Comment by spelexander** [comment]
+1

**Comment by fabiobiondi** [comment]
Hi guys, thank you for your interest.
I have used it in a personal project but I have never released it on GitHub because of some minor bugs in some scenarios that I had no time to fix and honestly because I wasn't so proud of that code :D

I have developed it with D3.js version 3 and after a couple of months version 4 and 5 were released. But I had no time to study them as well. 
So now you know the reason I'm not sharing my code :)

**Comment by lonewolfaka9** [comment]
post the code and we might fix it :-(

**Comment by fabiobiondi** [comment]
OK. It's on my old Mac. When I'll come back to home i'll post it ( 10 days ).
Sorry guys :)

**Comment by LEADER01** [comment]
Hello, any chance you'll still be able to grab code from your old Mac? :upside_down_face:

**Comment by np3xpu** [comment]
Did you ever get home?

**Comment by lonewolfaka9** [comment]
i think best way to replicate this is to create a   donut chart using d3 and inset icons into it & make them clickable

https://stackoverflow.com/questions/26690852/is-it-possible-to-insert-an-icon-in-the-arc-of-a-donut-chart-with-d3 inserting icons
http://jsfiddle.net/preciousmind/y3of6u7q/  handling click or mouseover event

**Comment by devdev-dev** [comment]
Did you find your old Mac? And more important did you find the source code?
